wp/resources/sample_binaries/pointer_flag/run_wp_sat.sh

```diff
@@ -0,0 +1,23 @@
+# This is a test that tests that the wp-pointer-reg-list flag works as expected.
+# Specifically, the function foo tries to use differing
+# return addresses that are pushed onto the stack to cause a different return
+# value between functions. The abscence of the flag should allow the pointer
```

Comment:
It's spelled "absence". Not to worry, my spelling is atrocious, but you might want to run some quick spell checks, or enable the squigglies in your IDE.